src/eko/kernels/evolution_integrals_qed.py

```diff
@@ -0,0 +1,303 @@
+r"""Compute evolution integrals needed for QED."""
```

Comment:
This is sharing too much code with its QCD counterpart! I believe you can abstract the function there (e.g. write a function `j00` that takes beta0 as an argument and that can be used by both side